docs/skopeo-list-tags.1.md

```diff
@@ -121,8 +124,23 @@ $ skopeo list-tags docker://localhost:5000/fedora
 
 ```
 
+### Docker-archive Transport
+
+To list the tags in a local docker-archive file:
+
+```sh
+skopeo list-tags docker-archive:/tmp/busybox.tar.gz
+{
+    "Repository": "",
```

Comment:
Hum, should this field be filled in? Or hidden if empty?

I guess we can start with the latter, and fill the field in later if we find a need to do so.

Reply:
> Hum, should this field be filled in? Or hidden if empty?
> 
> I guess we can start with the latter, and fill the field in later if we find a need to do so.
may be there are more images in the file
```
$ ./bin/skopeo list-tags  docker-archive:docker-two-images.tar.xz
{
    "Repository": "",
    "Tags": [
        "example.com/empty:latest",
        "example.com/empty/but:different"
    ]
}
```

Reply:
What I meant is that I think the output should be
```
$ ./bin/skopeo list-tags  docker-archive:docker-two-images.tar.xz
{
    "Tags": [
        "example.com/empty:latest",
        "example.com/empty/but:different"
    ]
}
```
without the pointless `Repository` line. That can be achieved using a `json:",omitempty"` annotation.